oauth2_http/java/com/google/auth/oauth2/JwtCredentials.java

```diff
@@ -34,6 +34,7 @@
 import com.google.api.client.json.webtoken.JsonWebSignature;
 import com.google.api.client.json.webtoken.JsonWebToken;
 import com.google.api.client.util.Clock;
+import com.google.api.client.util.GenericData;
```

Comment:
why is this here if there are no other changes in this class?

Reply:
Leftover from some experimentation I was doing before, removed.